### ISSUE TITLE
Proxy option

### ISSUE THREAD

**Issue by PeterPan123456** [shazamio/ShazamIO#65]
I'm having trouble using the library in one of my projects. I'm constantly getting rate limited by Shazam and would like to know if there is a way to use proxies, if support for httpx or something else could be implemented?

**Comment by dotX12** [comment]
@PeterPan123456, You can make a PR adding this functionality and I'll accept it :)

```python3
# example.py
proxy = 'http://your_user:your_password@your_proxy_url:your_proxy_port'
out = await shazam.recognize_song("data/dora.ogg", proxy=proxy)
```


```python3

**Comment by dotX12** [comment]
Closed, problem fixed in 0.5.0.

https://github.com/shazamio/ShazamIO/releases/tag/0.5.0
